content/articles/2018-07-broom-0-5-0.markdown

```diff
@@ -0,0 +1,169 @@
+
+## Tibble output
+
+All tidiers should now return `tibble`s rather than `data.frame`s. This allows broom to take advantage of the nice tibble print method and the more consistent behavior of tibbles:
+
+
+```r
+library(broom)
+
+fit <- lm(mpg ~ ., mtcars)
+tidy(fit)
+```
+
+```
+## # A tibble: 11 x 5
+##   term        estimate std.error statistic p.value
+##   <chr>          <dbl>     <dbl>     <dbl>   <dbl>
+## 1 (Intercept)  12.3      18.7        0.657   0.518
+## 2 cyl          -0.111     1.05      -0.107   0.916
+## 3 disp          0.0133    0.0179     0.747   0.463
+## 4 hp           -0.0215    0.0218    -0.987   0.335
+## 5 drat          0.787     1.64       0.481   0.635
+## # ... with 6 more rows
+```
+
+
+These changes will mostly likely affect you when you:
+
+- subset with `[`, which always returns a tibble.
+- set rownames on a tibble, which is deprecated.
+- use augment methods on models with matrix covariates specified in a formula, which will error.
+
+`augment()` will error with matrix covariates because tibbles are more strict about their contents than data frames. Data frames allow users to specify columns in a matrix, like so:
+
+
+```r
+y <- rnorm(5)
+x <- matrix(rnorm(10), nrow = 5)
+df <- data.frame(x, y)
+```
+
+Tibbles do not:
+
+
+```r
+library(tibble)
+
+tibble(x, y)
+```
+
+```
+## Error: Column `x` must be a 1d atomic vector or a list
+```
+
+Modelling functions will occasionally create a data frame like this, but since the model frame can't be coerced a tibble method, `augment()` will fail:
+
+
+```r
+fit <- lm(y ~ x, df)
+augment(fit)
+```
+
+```
+## Error: Column `x` must be a 1d atomic vector or a list
+```
+
+In some cases, explicitly passing the original dataset via the `data` argument can resolve this:
+
+
+```r
+augment(fit, data = df)
+```
+
+```
+## # A tibble: 5 x 10
+##        X1     X2      y .fitted .se.fit  .resid  .hat .sigma .cooksd
+## *   <dbl>  <dbl>  <dbl>   <dbl>   <dbl>   <dbl> <dbl>  <dbl>   <dbl>
+## 1 -0.479  -0.863  0.367  -0.505   0.437  0.871  0.244 0.752   0.137 
+## 2 -0.621  -1.54  -0.448  -0.594   0.591  0.146  0.446 1.24    0.0132
+## 3 -0.0472 -0.872 -1.86   -1.02    0.560 -0.837  0.400 0.633   0.330 
+## 4 -1.33   -0.229  0.474   0.753   0.863 -0.279  0.950 0.0260 12.8   
+## 5  0.995   1.85  -1.13   -1.22    0.868  0.0986 0.960 1.15    2.52  
+## # ... with 1 more variable: .std.resid <dbl>
+```
+
+Support for matrix-columns is on the way in dplyr and in a release cycle or two this won't be an issue.
+
+Deprecated tidiers still return data frames. Tidiers for mixed models also return data frames; more on this below.
+
+## New infrastructure for testing and documentation
+
+We are heavily invested in making it easier to contribute to broom, and also in making broom behavior more standardized and consistent. To this end, we've written new testing and documentation infrastructure that automates many of these processes.
```

Comment:
Could we have one example? e.g. showing how you'd use `check_tidy()` in one line